Sources/Markdown/Walker/Walkers/MarkupFormatter.swift

```diff
@@ -345,8 +345,18 @@ public struct MarkupFormatter: MarkupWalker {
         /// The length of the last line.
         var lastLineLength = 0
 
-        /// The current line number.
+        /// The line number of the most recently printed content. This is updated in
+        /// `addressPendingNewlines(for:)` when line breaks are printed.
```

Comment:
Nit: Maybe this should be in the discussion section?

```suggestion
        /// The line number of the most recently printed content. 
        ///
        /// This is updated in `addressPendingNewlines(for:)` when line breaks are printed.
```

Reply:
I like this idea! I went ahead and applied your suggestion.